### ISSUE TITLE
Great checks, but not super necessary

### ISSUE THREAD

**Issue by jmcrey** [EJGM1/pwp-capstones#1]
https://github.com/EJGM1/pwp-capstones/blob/ec1c32b07f7882bc2e21c5246e4966a22804fdc1/TomeRater%20(2)/TomeRater.py#L3-L12

So, I love the fact that we are validating the variable types here. It really shows that the code is detail-oriented and considers every aspect of implementation. There a couple problems here though. First, the checks aren't super necessary. Honestly, if the user is initiating the class wrong, that is not the fault of the programmer, it is the fault of the user. In my opinion, it is better to let the functions hard fail than to catch a `TypeError`. However, some people don't agree with me, which brings me to my second point. 

Second, if we want to enforce proper types, then we should be using the `raise` function to make sure that the function errors out and the structure is not stored in memory. Using `raise` will make the function throw an error so the user will know exactly what went wrong. In this case, the function should `raise` a `TypeError`. Here is an example:

```
if type(name) == str: 
     self.name = name 
 else: 
     raise TypeError 
```

This will display a message to the user that there has been an incorrect type assigned, and that they should change it. Of course, we can always make the message more descriptive by creating a custom exception, such as `StrTypeError`, but that might be too granular. In this case, `TypeError` is perfectly fine to throw. If you have any questions about why `raise` is a better approach thing simply printing, please ask! I am happy to explain further.

In summary, I love the fact that the code is taking the time to catch the type. However, it should extend this functionality to ensure that the structure throws an error OR it should let the functions hard fail at an uncaught exception if the user assigns each variable the incorrect typing.
